src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -2230,6 +2233,11 @@ public class ComputerInfo
         /// The Windows ReleaseId, from the Windows Registry.
         /// </summary>
         public string WindowsVersion { get; internal set; }
+
+        /// <summary>
+        /// The Windows UBR, from the Windows Registry.
```

Comment:
Maybe better `The Windows UBR (Update Build Revision), from the Windows Registry. `